.ci/travis/openjdk14-test/jdk14-test-excluded-files.list

```diff
@@ -0,0 +1,28 @@
+
+## This file contains all compilable files from
+## https://download.java.net/openjdk/testresults/14/archives/36/langtools-36-summary.txt
+## that Checkstyle cannot parse. When adding new files to exclude, make sure to create an issue
+## and link to it in this file so that we can track it. Once the issue is resolved, please remove
+## the file from this list to prove that Checkstyle can parse it.
+
+jdk14/test/langtools/jdk/javadoc/tool/T4994049/FileWithTabs.java
+jdk14/test/langtools/tools/javac/MethodParameters/UncommonParamNames.java
+jdk14/test/langtools/tools/javac/TextBlockLang.java
+jdk14/test/langtools/tools/javac/annotations/typeAnnotations/8013180/QualifiedName.java
+jdk14/test/langtools/tools/javac/annotations/typeAnnotations/failures/CheckErrorsForSource7.java
+jdk14/test/langtools/tools/javac/annotations/typeAnnotations/failures/T8011722.java
+jdk14/test/langtools/tools/javac/annotations/typeAnnotations/newlocations/AllLocations.java
+jdk14/test/langtools/tools/javac/annotations/typeAnnotations/newlocations/NestedTypes.java
+jdk14/test/langtools/tools/javac/enum/TrailingComma.java
+jdk14/test/langtools/tools/javac/lambda/speculative/InferStrict.java
+
+## Related to https://github.com/checkstyle/checkstyle/issues/8291 ##
```

Comment:
can you create issues on other cases ?

Reply:
https://github.com/checkstyle/checkstyle/issues/8650
https://github.com/checkstyle/checkstyle/issues/8651
https://github.com/checkstyle/checkstyle/issues/8652
https://github.com/checkstyle/checkstyle/issues/8653
https://github.com/checkstyle/checkstyle/issues/8654
https://github.com/checkstyle/checkstyle/issues/8655
https://github.com/checkstyle/checkstyle/issues/8656